ethereum/gatekeeper-cli/README.md

```diff
@@ -104,58 +104,49 @@ USAGE
 * [`gateway-eth rename-gatekeeper-network ID NAME`](#gateway-eth-rename-gatekeeper-network-id-name)
 * [`gateway-eth revoke ADDRESS`](#gateway-eth-revoke-address)
 * [`gateway-eth unfreeze ADDRESS`](#gateway-eth-unfreeze-address)
-* [`gateway-eth verify ADDRESS`](#gateway-eth-verify-address)
 
 ## `gateway-eth add-gatekeeper ADDRESS`
 
 Add a gatekeeper to a gatekeeper network
 
 ```
 USAGE
-  $ gateway-eth add-gatekeeper [ADDRESS] [-h] [-p <value>] [-t <value>] [-n <value>] [-c
```

Comment:
Dunno why all these readme changes...maybe the format output by `manifest` changed in a recent oclif version?